cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -31,6 +31,11 @@ options:
       privileged mode. If "auto", kube-apiserver will not run in privileged
       mode by default, but will switch to privileged mode if gpu hardware is
       detected on a worker node.
+  automatic-nvidia-plugin-loading:
+    type: boolean
+    default: True
+    description: |
+      Automatically load the nvidia device plugin daemonset.
```

Comment:
I'd feel safer with something like `enable-nvidia-plugin=auto`. Even if we only allow `false` or `auto` today, we might want to allow `true` in the future to override the automatic behavior. It'd be harder to grow into that with the boolean `automatic-nvidia-plugin-loading` config proposed here.

Reply:
Easy!